### PR TITLE
feat: Added ability to toggle scrolling of modal content

### PR DESCRIPTION
**Pull request by cclinard84**
Without fix

image

With fix

image

### CODE REVIEW

File: addon/components/nrg-modal/component.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ export default Component.extend({
   basic: false,
   sidebar: false,
   lightbox: false,
+  scrolling: true,
```
Comment (zachnthebox) [inline addon/components/nrg-modal/component.js:24]: Should this be a `reads` value of `!this.renderInPlace && !this.masterDetail && !this.sidebar && !this.lightbox` instead?

File: addon/components/nrg-modal-container/modal-wrapper/component.js [hunk 2]
```diff
@@ -89,7 +90,7 @@ export default Component.extend(ResizeMixin, {
       if (!this.sidebar) {
         appliedClasses.push('content');
       }
-      if (!this.renderInPlace && !this.masterDetail && !this.sidebar && !this.lightbox) {
+      if (!this.renderInPlace && !this.masterDetail && !this.sidebar && !this.lightbox && this.scrolling) {
```
Comment (zachnthebox) [inline addon/components/nrg-modal-container/modal-wrapper/component.js:93]: If you just add it here, you don't have full controller over the scrolling state as you one.

File: addon/components/nrg-modal-container/modal-wrapper/component.js [hunk 3]
```diff
@@ -28,8 +28,14 @@ export default Component.extend(ResizeMixin, {
   lightbox: alias('modal.lightbox'),
   modalClass: alias('modal.modalClass'),
   dimmerClass: alias('modal.dimmerClass'),
+  scrolling: alias('modal.scrolling'),
 
+  notMasterDetail: not('masterDetail'),
+  notLightbox: not('ligthbox'),
   notSidebar: not('sidebar'),
+  notRenderInPlace: not('renderInPlace'),
+  notSecondaryModalUse: and('notMasterDetail', 'notLightbox', 'notSidebar', 'notRenderInPlace'),
```
Comment (zachnthebox) [inline addon/components/nrg-modal-container/modal-wrapper/component.js:37]: This should be in the modal itself and not the wrapper..


```js
scrolling: and('notMasterDetail', 'notLightbox', 'notSidebar', 'notRenderInPlace'),
```

Comment (cclinard84) [reply addon/components/nrg-modal-container/modal-wrapper/component.js:37]: Ahh I see what you were thinking now. Thanks!

**Review by zachnthebox — APPROVED** [review]

**Pull request merged**